tests/telemetry/test_telemetry.py

```diff
@@ -458,6 +464,72 @@ def test_validate_entries(monkeypatch):
     assert res == (event_id, uid, action, client_time, elapsed_time)
 
 
+def test_get_latest_version(monkeypatch):
+    is_latest = telemetry.get_latest_version()
+    assert isinstance(is_latest, str)
+    version_index = [i for i, ltr in enumerate(is_latest) if ltr == '.']
+    assert len(version_index) == 2
+
+    mock_httplib = Mock()
+    mock_httplib.HTTPSConnection().request.side_effect = Exception
+    monkeypatch.setattr(telemetry, 'httplib', mock_httplib)
+    is_latest = telemetry.get_latest_version()
+    assert is_latest == ploomber.__version__
+
+    # Mock version and the conf, check it produces the same version
+
+
+def write_to_conf_file(path, version, last_check):
+    (path).write_text("stats_enabled: True\n"
+                      f"last_version_check: {last_check}\n"
+                      f"latest_ploomber_version: {version}\n")
+
+
+def test_check_version(tmp_directory, monkeypatch):
+    # Path conf file
+    stats = Path('stats')
+    stats.mkdir()
+    path = stats / 'config.yaml'
+    monkeypatch.setattr(telemetry, 'DEFAULT_HOME_DIR', '.')
+    monkeypatch.setattr(ploomber, '__version__', '0.14.8')
+    # (path).write_text("stats_enabled: True\n"
+    #                   f"last_version_check: 2022-01-20 10:51:41.082376\n"
+    #                   f"latest_ploomber_version: 0.14.8\n")
+    write_to_conf_file(path=path,
+                       version='0.14.8',
+                       last_check='2022-01-20 10:51:41.082376')
+
+    # Test no warning when same version encountered
+    # with warnings.catch_warnings():
+    #     warnings.filterwarnings("error")
```

Comment:
remove all commented code

Reply:
So this is commented since I couldn't mock the __version__ to a specific version for some reason, I'm not sure what's going on there